frontend/src/views/Apps/TierSelection/FreeTierModal.js

```diff
@@ -17,42 +21,38 @@ class FreeTierModal extends Component {
         <Modal.Body>
-          <h4>How the free tier works?</h4>
+        <h4>HOW IT WORKS? - LAUNCH OFFERING</h4>
           <p>
-            The free tier is a limited time infrastructure plan for any app to
-            receive free API throughput maxed at 1 Million Relays per Day.
+            In the Mainnet Launch Offering, Pocket Network Inc stakes on behalf of the customer and manages
+            the staked POKT as well as the AAT(Application Authentication Token). This will allow any app to access
+            the network and connect to any of the available chains for free.
           </p>
           <p>
-            In the free tier, Pocket Network Inc stakes on behalf of the
-            customer and manages the staked POKT as well as the AAT (Application
-            Authentication Token). This will allow any app to access the network
-            and connect to any of the available chains for free.
-          </p>
-          <p>
-            As your app grows and scales you will be able to unstake and
-            transition to the Custom Tier to get the exact amount of relay
-            throughput you need.
-          </p>
-          <p>
-            If you still have questions please take a look at our{" "}
-            <a href="/todo">FAQ.</a>
+            Once your app scales up you can unstake and transition to the Stake and Scale plan to get the exact amount of
+            throughput you need. If you still have questions take a loot at our <a style={{fontWeight: "300", textDecoration: "none"}} href="https://dashboard.docs.pokt.network/docs/faq">FAQ</a>.
           </p>
           <p className="subtitle">
             <b>Pocket Network</b>
           </p>
           <p>
-            In the free tier, Pocket Network Inc stakes on behalf of the user
-            and manages both the staked POKT and Application Authentication
-            Token (AAT). PNI reserves the right to revoke relay throughput at
-            any time for violating the Terms and Conditions. PNI is not
-            responsible for damage resulting from managing AATs or POKT.
+            In the Mainnet Launch Offering, Pocket Network Inc stakes on behalf of the app and manages the staked POKT and Application
+            Authentication Token (AAT). PNI reserves the right to revoke throughput at any time for violation of the <Link style={{fontWeight: "300", textDecoration: "none"}} to={ROUTE_PATHS.termsOfService}>Terms of Use</Link>.
+          </p>
+          <ul className="violation-list">
+            <li>Cases that may result in revocation of AAT include.</li>
+            <li>Usage falls below 50,000 daily relays for over 3 months.</li>
+            <li>User is deemed to be using a fraudulent application or attacking the network.</li>
+            <li>User is exploiting the offering to run multiple free accounts.</li>
+            <li>After 1 Year of service, PNI will revisit continuning this service for all.</li>
```

Comment:
This should be moved below the list into the next paragraph as it is not the same as the other list items.

"After one year of service,..."